selftests/functional/test_basic.py

```diff
@@ -715,8 +720,10 @@ def test_kill_stopped_sleep(self):
                               % (self.tmpdir, SLEEP_BINARY))
         proc.read_until_output_matches(["\(1/1\)"], timeout=3,
                                        internal_timeout=0.01)
-        # We need pid of the avocado, not the shell executing it
-        pid = int(process.get_children_pids(proc.get_pid())[0])
+        # We need pid of the avocado process, not the shell executing it
+        avocado_shell = psutil.Process(proc.get_pid())
+        avocado_proc = avocado_shell.children()[0]
+        pid = avocado_proc.pid
```

Comment:
Might I ask what is the problem with `process.get_children_pids`? Is it not mac-compatible? If so than a better fix would be to make `process.get_children_pids` mac-compatible and keep using it here.